packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts

```diff
@@ -109,6 +110,7 @@ export const defaultLivenessDisplayText: Required<LivenessDisplayText> = {
   hintCanNotIdentifyText: 'Move face in front of camera',
   hintTooCloseText: 'Move back',
   hintTooFarText: 'Move closer',
+  /** @deprecated String is no longer in use */
```

Comment:
Same question as above
```suggestion
 /** @deprecated `hintHoldFacePositionCountdownText` is no longer in use and will be removed in a future major version release. */
```